code/_onclick/hud/alert.dm

```diff
@@ -845,63 +871,39 @@ or shoot a gun to move around via Newton's 3rd Law of Motion."
 		if(timeleft <= 0)
 			return PROCESS_KILL
 		cut_overlay(time_left_overlay)
-		var/obj/abstract_object = new
-		abstract_object.maptext = "<span style='font-family: \"Small Fonts\"; font-weight: bold; font-size: 8px; color: [(timeleft <= 10 SECONDS) ? "red" : "white"]; -dm-text-outline: 1px black'>[CEILING(timeleft / 10, 1)]</span>"
-		var/matrix/move_matrix = new
-		move_matrix.Translate(4, 16)
-		abstract_object.transform = move_matrix
-		time_left_overlay = image(abstract_object)
-		time_left_overlay.plane = plane
+		time_left_overlay = new
+		time_left_overlay.maptext = MAPTEXT("<span style='font-weight: bold; color: [(timeleft <= 10 SECONDS) ? "red" : "white"]; -dm-text-outline: 1px black'>[CEILING(timeleft / 10, 1)]</span>")
```

Comment:
You don't need the text outline, MAPTEXT does that for you. Also use `b` instead of changing font style.

Reply:
Double check tho